examples/lock-app/nrfconnect/main/AppTask.cpp

```diff
@@ -179,7 +179,7 @@ CHIP_ERROR AppTask::Init()
 
 #ifdef CONFIG_MCUMGR_SMP_BT
     // Initialize DFU over SMP
-    GetDFUOverSMP().Init(RequestSMPAdvertisingStart);
```

Comment:
Why do we only change this in lock-app? We have the same code in other samples as well.

Reply:
Sure, I will apply this change to other examples once the API is approved.